content/docs/14-troubleshooting/30-pcg.md

```diff
@@ -88,37 +103,220 @@ When the PCG installer instance is powered on, it goes through a bootstrap proce
+  |`Failed to deploy image: Failed to create govomiClient`| [Scenario - Failed to Deploy Image](#scenario---failed-to-deploy-image) |
+  |`No route to host`| [Scenario - No Route to the Kube API Server](#scenario---no-route-to-the-kube-api-server) |
+
+
+6. Verify the following cloud gateway configurations in Palette.  
+
+    - Check the VMware vCenter server field. The field expects a URL or an IP address. If you use a URL, do not place a preceding "http://" or "https://" string. Also, select the **Use Qualified Network Name** checkbox if you use a URL. The screenshot below displays the vCenter server field you configure in Palette. 
+    ![A screenshot displaying the vCenter server field you configure in Palette](/troubleshooting-pcg-cluster_config_1.png)
+
+    - Ensure the VMware cloud properties are specified correctly in the cloud gateway configuration. You must use the vSphere data center and the folder where you have permission to create resources. 
+
+    - If you choose the DHCP option, enable the Dynamic DNS in your DNS server.
+
+    - Ensure the search domain is correctly defined in the fault domain's DNS settings. The screenshot below displays the VMware cloud properties you configure in Palette. 
+    ![A screenshot displaying the VMware cloud properties you configure in Palette](/troubleshooting-pcg-cluster_config_2.png)
 
-The installation of a gateway cluster may encounter errors or get stuck in the provisioning state due to such reasons as a lack of infrastructure resources, unavailable IP addresses, or the inability to perform a Network Time Protocol (NTP) sync.
 
-While these are the most common errors, some other issues might be related to the underlying VMware environment. 
+7. Suppose the orchestration is stalled due to the incorrect cloud gateway configuration. In that case, you must delete the gateway and reconfigure it again.
+
+
+8. If the problem persists, download the cluster logs from Palette. The screenshot below will help you locate the button to download logs from the cluster details page. 
+
+  ![A screenshot highlighting how to download cluster logs in from Palette.](/troubleshooting-pcg-download_logs.png)
+
+
+9. Share the logs with Spectro Cloud's support team at [support@spectrocloud.com](mailto:support@spectrocloud.com).
+<br />
+
+
+# Scenario - No Progress After Creating the Container Manager
+Suppose the PCG events in Palette display no progress after the specific event, `Created container manager`. 
+
+The issue can occur when the gateway installer VM fails to connect to the Spectro Cloud API, [https://api.spectrocloud.com](https://api.spectrocloud.com), to download the installation artifacts. Another potential reason is that the gateway installer may not have the required permissions to store the installation artifacts in the **spectro-templates** folder. The installer downloads the images for the worker nodes and stores them in the **spectro-templates** folder during the cluster provisioning.
```

Comment:
```suggestion
This issue can occur when the gateway installer VM fails to connect to the Palette API endpoint and download the installation artifacts. Another potential reason is that the gateway installer may not have the required permissions to store the installation artifacts in the **spectro-templates** folder. The installer downloads the images for the worker nodes and stores them in the **spectro-templates** folder during the cluster provisioning.
```
We don't have to make the API a markdown URL. If you click on the API URL you'll get an error.